python/tests/aggregate_tests/test_varcharn_count.py

```diff
@@ -0,0 +1,55 @@
+from .aggtst_base import TstView
+
+
+class aggtst_atbl_varcharn(TstView):
+    def __init__(self):
+        # Validated on Postgres
+        self.data = [
+            {"id": 0, "f_c1": None, "f_c2": "abc  "},
```

Comment:
so postgres also has spaces here?

Reply:
Yes. They both have a single space after "c".

Reply:
I see two spaces

Reply:
Yes. Sorry. They both have two spaces after "c". 

This is the result from Postgres. 

"null"	"abc  "
"hello"	"fred"
"@abc"	"varia"
"hello"	"examp"